README.md

```diff
@@ -469,10 +470,13 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Haystack](https://github.com/deepset-ai/haystack) ![](https://img.shields.io/github/stars/deepset-ai/haystack.svg?style=social) - Haystack is an open source NLP framework to interact with your data using Transformer models and LLMs (GPT-3 and alike). Haystack offers production-ready tools to quickly build ChatGPT-like question answering, semantic search, text generation, and more.
 * [Kashgari](https://github.com/BrikerMan/Kashgari) ![](https://img.shields.io/github/stars/BrikerMan/Kashgari.svg?style=social) - Kashgari is a simple and powerful NLP Transfer learning framework, build a state-of-art model in 5 minutes for named entity recognition (NER), part-of-speech tagging (PoS), and text classification tasks.
 * [LangChain](https://github.com/hwchase17/langchain) ![](https://img.shields.io/github/stars/hwchase17/langchain.svg?style=social) - LangChain assists in building applications with LLMs through composability.
+* [NLTK](https://github.com/nltk/nltk) ![](https://img.shields.io/github/stars/nltk/nltk.svg?style=social) - NLTK (Natural Language Toolkit) is a suite of open source Python modules, data sets, and tutorials supporting research and development in Natural Language Processing.
 * [OpenAI GPT-2](https://github.com/openai/gpt-2) ![](https://img.shields.io/github/stars/openai/gpt-2.svg?style=social) - OpenAI's code from their paper ["Language Models are Unsupervised Multitask Learners"](https://d4mucfpksywv.cloudfront.net/better-language-models/language-models.pdf).
+* [refinery](https://github.com/code-kern-ai/refinery) ![](https://img.shields.io/github/stars/code-kern-ai/refinery.svg?style=social) - The data scientist's open-source choice to scale, assess and maintain natural language data.
```

Comment:
This seems also more relevant for data labelling section

Reply:
That is a good question, @axsaucedo I am thinking about the current solo categorization in the current structure of our list since we are not allowed to attribute more than one label to a single tool as indicated by https://github.com/EthicalML/awesome-production-machine-learning/issues/383.

As far as I understand, if a tool is designed for data labeling, it should be universally used for data labeling, which means, ideally practitioners, can use it to label their data regardless of their working domain.

On the other hand, if a tool only caters to a specific domain, such as the NLP dataset. It is a domain tool that is built exclusively for NLP practitioners (since other practitioners cannot use it for other labeling purpoess). Then we should categorize it as an NLP tools rather than a general data labeling tool. What's your idea? @axsaucedo

Reply:
Looking at the repo it does seem better fit for data labelling than general NLP as invovles dataset mgmt as well

Reply:
> Looking at the repo it does seem better fit for data labelling than general NLP as invovles dataset mgmt as well

I know this tool has the functionalities of dataset management but it is exclusively focused on NLP dataset, for example, we cannot use this tool to create/manage/visualize other modal datasets such as images, audio, code, etc... @axsaucedo

Reply:
IMO that's ok, what you suggest is just multimodal data labelling, it's ok for datalabelling being single modality

Reply:
> IMO that's ok, what you suggest is just multimodal data labelling, it's ok for datalabelling being single modality
I think it is better to discuss it over here: https://github.com/EthicalML/awesome-production-machine-learning/issues/387
I found a lot of times this kind issue make a dilemma for the contributors to decide which categories to choose for a new tool. @axsaucedo

Reply:
For sure, if you move it to data labelling we can continue discussing in issue, alternative option is to remove from PR

Reply:
I think it is better to know what makes things right and then do the right thing, rather than make a detour. I have proposed some issues but nobody discussed them at all. If so, why should I make the issues? @axsaucedo